src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +76,16 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Alert to using bootstrap's push / pull in Clarity grid
+          if (updated.search(/push-(xs|sm|md|lg|xl)-([0-9])/g) > -1) {
```

Comment:
We also need a check for `pull-` classes.

Reply:
doh.. you are right!